integration/networktest/log.go

```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/obscuronet/go-obscuro/integration/common/testlog"
 )
 
-// EnsureTestLogsSetUp calls Setup iff it hasn't already been called (some tests run tests within themselves, we don't want
+// EnsureTestLogsSetUp calls Setup if it hasn't already been called (some tests run tests within themselves, we don't want
```

Comment:
That wasn't a typo, "iff" is shorthand for "if and only if" but tbf we should probably avoid that if it's not widely used so I don't mind the edit.